service/matching/liveness_test.go

```diff
@@ -118,7 +117,7 @@ func (s *livenessSuite) TestEventLoop_Shutdown() {
 	liveness := newLiveness(s.timeSource, s.ttl, func() { atomic.CompareAndSwapInt32(&s.shutdownFlag, 0, 1) })
 	liveness.Start()
 
-	s.timeSource.Update(time.Now().Add(s.ttl * 4))
+	s.timeSource.Advance(s.ttl * 4)
 	<-liveness.shutdownChan
 	s.Equal(int32(1), atomic.LoadInt32(&s.shutdownFlag))
```

Comment:
hm :\  this is a logical race, since the chan is closed before the shutdown func is called.

not related to this PR / no need to fix here tho.